core/src/main/java/com/datastax/oss/driver/internal/core/cql/DefaultPreparedStatement.java

```diff
@@ -103,7 +103,6 @@ public DefaultPreparedStatement(
     this.consistencyLevelForBoundStatements = consistencyLevelForBoundStatements;
     this.serialConsistencyLevelForBoundStatements = serialConsistencyLevelForBoundStatements;
     this.areBoundStatementsTracing = areBoundStatementsTracing;
-
     this.codecRegistry = codecRegistry;
     this.protocolVersion = protocolVersion;
```

Comment:
Nit: could you avoid modifying unrelated lines? This adds noise when you do a git blame on the file later (admittedly not for deleted lines, but still).